cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,26 @@
+class PaymentStepPage{
+
+    private bankWire: string;
+    private confirmPay: string;
+    private finishShopping: string;
+
+    constructor(){
+        this.bankWire = "[class=bankwire]";
+        this.confirmPay = "#cart_navigation > button";
+        this.finishShopping = "#center_column > div > p > strong";
+    }
+
+    public payingWire(): void {
+        cy.get(this.bankWire).click()
+    }
+
+    public confirmingPay(): void {
+        cy.get(this.confirmPay).click()
+    }
+
+    public finishingShopping():void {
```

Comment:
A best practice is to not perform assertions on page objects, so, it will be better to make this method return the confirmation title, change the name of it as getConfirmationTitle or something like that, and perform the assertion on the test, that is the cypress/integration/buy-shirt.spec.ts file